praktikumsplaner-frontend/.env

```
@@ -0,0 +1,8 @@
+#see https://git.muenchen.de/km23/mucatar (internal only), otherwise use own hosted instance url of https://github.com/it-at-m/ad2image
+VITE_AD2IMAGE_URL="https://mucatar.muenchen.de/"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)